mempool/reactor.go

```diff
@@ -641,6 +649,6 @@ func (rc *redundancyControl) blockHaveTx() {
 }
 
 func (rc *redundancyControl) triggerAdjustment() {
-	<-rc.adjustTicker.C
+	rc.triggerAdjustmentCh <- struct{}{}
```

Comment:
This new channel is not needed. It can be replaced by calling `rc.adjustRedundancy(memR)` directly. What I'm wondering is if it's also required to do `rc.adjustTicker.Reset(rc.adjustInterval)` afterwards. Because if two `adjustRedundancy` are triggered at the same time, the second one that executes will have zero received transactions.

Reply:
YOu are right, I will replace this.. The reset would be needed in my opinion because you want to be sure the interval is respected afterwards.